consensus/istanbul/core/preprepare.go

```diff
@@ -98,27 +122,12 @@ func (c *core) handlePreprepare(msg *message, src istanbul.Validator) error {
 		return err
 	}
 
-	// Here is about to accept the PRE-PREPARE
+	// TODO(asa): Can we skip to COMMIT if we have a PREPARED certificate already?
```

Comment:
Wdyt @trianglesphere?

Reply:
Not sure. My gut feeling is no, but I'd need some time to know either way. Feels out of scope for now.